src/tests/performance/prio_test.rs

```diff
@@ -0,0 +1,313 @@
+const REQUESTS: u16 = 300;
+const RESPONSE_TIMEOUT: Duration = Duration::from_secs(3);
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 8)]
+#[allow(non_snake_case)]
+async fn p002_t1_PRIO_MSG_latency() {
+    // ZG-PERFORMANCE-002, Block getting latency while malicious nodes send higher priority msgs
+    //
+    // We test the overall performance of a node's get blocks (with certs) latency while other
+    // malicious nodes send higher priority messages.
+    //
+    // Test should be inspected manually to check how other malicious nodes affect the latency of
+    // the node under test.
+    //
+    // Sample results:
+    // ┌─────────┬───────────────────┬────────────┬────────────┬────────────┬────────────────┬────────────────┬────────────┐
+    // │  peers  │  malicious peers  │  requests  │  min (ms)  │  max (ms)  │  std dev (ms)  │  completion %  │  time (s)  │
+    // ├─────────┼───────────────────┼────────────┼────────────┼────────────┼────────────────┼────────────────┼────────────┤
+    // │       1 │                 1 │        300 │          1 │          1 │              0 │         100.00 │       0.41 │
+    // ├─────────┼───────────────────┼────────────┼────────────┼────────────┼────────────────┼────────────────┼────────────┤
+    // │       1 │                50 │        300 │          1 │          2 │              1 │         100.00 │       0.51 │
+    // ├─────────┼───────────────────┼────────────┼────────────┼────────────┼────────────────┼────────────────┼────────────┤
+    // │       1 │               100 │        300 │          1 │          3 │              1 │         100.00 │       0.64 │
+    // ├─────────┼───────────────────┼────────────┼────────────┼────────────┼────────────────┼────────────────┼────────────┤
+    // │       1 │               200 │        300 │          1 │          5 │              1 │         100.00 │       0.62 │
+    // ├─────────┼───────────────────┼────────────┼────────────┼────────────┼────────────────┼────────────────┼────────────┤
+    // │       1 │               300 │        300 │          1 │         14 │              2 │         100.00 │       0.80 │
+    // ├─────────┼───────────────────┼────────────┼────────────┼────────────┼────────────────┼────────────────┼────────────┤
+    // │       1 │               400 │        300 │          1 │          9 │              1 │         100.00 │       0.82 │
+    // └─────────┴───────────────────┴────────────┴────────────┴────────────┴────────────────┴────────────────┴────────────┘
+    // *NOTE* run with `cargo test --release  tests::performance::prio -- --nocapture`
+    // Before running test generate dummy devices with different ips using toos/ips.py
+
+    let synth_counts = vec![1, 50, 100, 200, 300, 400];
+
+    let mut table = RequestsTable::default();
+
+    for synth_count in synth_counts {
+        // synth_count malicious tasks plus one normal synth_node
+        let barrier = Arc::new(Barrier::new(synth_count + 1));

```

Comment:
maybe, jsut maybe, this wouldn't be a bad approach:

```rust
let hp_synth_cnt = synth_count;
let lp_synth_cnt = 1;  // or a const somewhere
let total_synth_cnt = hp_synth_cnt + lp_synth_cnt;
```
then we could use these instead of doing `synth_node + 1` in the code below.

----

and then in the table below:
```rust
                table.add_row(RequestStats::new(
                    lp_synth_cnt,  // now we don't need those comments anymore
                    hp_synth_cnt,  // making my below comment also depracted
```